validations/renavam.js

```diff
@@ -6,8 +6,12 @@ export default (name, value) => {
     valid: false
   }
 
+  if (value.length < 8) return field
+
+  const totalLength = 11
   let renavam = utils.removeWhiteSpace(value)
-  
+  renavam = renavam.length < totalLength ? renavam.padStart(totalLength, '0') : renavam
```

Comment:
Aqui não há necessidade de atribuirmos o msm valor pra `renavam`, podemos simplificar o código assim

```js
if (renavam.length < totalLength)  renavam = renavam.padStart(totalLength, '0') 
```

ou sem usar `if` 

```js
(renavam.length < totalLength) && (renavam = renavam.padStart(totalLength, '0'))
```

Reply:
Ao invés de `for` talvez seja interessante usarmos `reduce`

```js
let digit = 0
for (var i = 0; i < 10; i++){
  digit += renavam[i] * pattern[i];
}
```

Para opção 1

```js
let digit = renavam.reduce((acc, cur, idx) => acc + cur * pattern[idx], 0)
```

Para opção 2

```js
let digit = [10].reduce((acc, cur, idx) => acc + renavam[idx] * pattern[idx], 0)
```